.gitlab-ci.yml

```diff
@@ -34,11 +34,11 @@ replay_tests:
   script:
     - timeout 10m ant replay-tests
 
-snapshot_tests:
-  stage: full-test
-  tags: [benchmarks, infinity]
-  script:
-    - timeout 10m ant snapshot-tests
+#snapshot_tests:
```

Comment:
Needs a comment, something that says why it is disabled, and when it will be enabled again (probably PR number. ideally, you could also add a todo on the related PR)